.up-env

```diff
@@ -1,4 +1,4 @@
-export TYPE="channel"
+export TYPE="source"
```

Comment:
Build seems to fail on `channel`: https://app.circleci.com/pipelines/github/algorand/py-algorand-sdk/711/workflows/f1d3e702-cf2a-40dc-ac8a-49731c2b6f36/jobs/2643

Reply:
The reason for failure wasn't obvious to me. Perhaps [this error on line 1218 of the build log](https://app.circleci.com/pipelines/github/algorand/py-algorand-sdk/711/workflows/f1d3e702-cf2a-40dc-ac8a-49731c2b6f36/jobs/2643?invite=true#step-102-1218) gives a clue?

This brings up a general question. Should we keep supporting the nightly channel mode?

I suspect we still want to. But maybe because of how slow it is, it could be carved out to nightly build. Worth further discussion.

Reply:
Good point - I'll try to investigate a little more today as well

Reply:
@algochoi Can you try merging in `master`?

I think https://github.com/algorand/algorand-sdk-testing/pull/207 addresses the 404 error.

Reply:
Nice, that fixes the build on my machine. Thanks!